src/main/groovy/com/ullink/gradle/opencover/OpenCover.groovy

```diff
@@ -36,7 +33,7 @@ class OpenCover extends ConventionTask {
             getTargetAssemblies()
         }
         outputs.files {
-            getCoverageReportPath()
+           getReportsFolder().listFiles()
```

Comment:
you can specify an output folder using `dir` instead of `files`:
https://docs.gradle.org/current/javadoc/org/gradle/api/tasks/TaskOutputs.html#dir(java.lang.Object)

Should be more accurate as it's difficult to tell when this list will be evaluated (e.g. before task execution, the folder is empty)

Reply:
Done